Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -61,6 +62,9 @@ void ComputeBattleUnitDefense(struct BattleUnit * attacker, struct BattleUnit *
         if (BattleSkillTester(defender, SID_SorceryBlade))
             status = def < res ? def : res;
 #endif
+
+        if (GetCombatArtInForce(&defender->unit) == CID_Detonate)
+            status = 0;
```

Comment:
The skill description does not mention to ignore the defense. the calculation process should also not be placed here.

Reply:
I mean, it's sort of implied? "Deal remaining uses as damage" rather than "Set attack to remaining uses". But duly noted, I'll fix